cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -569,6 +569,11 @@ private CassandraNode buildCassandraNode(Protos.Offer offer, boolean seed, Strin
         CassandraNode.Builder builder = CassandraNode.newBuilder()
                 .setHostname(offer.getHostname())
                 .setTargetRunState(CassandraNode.TargetRunState.RUN)
+                .addDataVolumes(
+                    DataVolume.newBuilder()
+                        .setPath(configuration.getDefaultConfigRole().getPreDefinedDataDirectory())
+                        .build()
+                )
```

Comment:
Also not sure whether we should add the directory configured via the env option `CASSANDRA_DATA_DIRECTORY` to `CassandraNode.dataVolumes`. Could omit this for now as it might cause some problems when we have non-ephemeral data directories provided by mesos itself.